src/api/appUsers.js

```diff
@@ -1,23 +1,69 @@
-import BaseAPI from './base';
-import http from '../utils/http';
+import { BaseAPI } from './base';
+import { http } from '../utils/http';
 
-export default class AppUsersAPI extends BaseAPI {
-  retrieve(userId, auth) {
-    const url = this.getFullURL('appUsers', userId);
+
+/**
+ * Init API properties
+ * @typedef InitProps
+ */
+
+/**
+ * @class AppUsersAPI
+ * @extends BaseAPI
+ */
+export class AppUsersAPI extends BaseAPI {
```

Comment:
This is admittedly a highly religious comment, but I like camel-casing acronyms that are 3 letters and longer, so `AppUsersAPI` --> `AppUsersApi`.  This prevents insanity like `XMLHttpRequest`.